pkg/ccl/changefeedccl/changefeed_job_info.go

```diff
@@ -25,6 +25,8 @@ const (
 const (
 	// resolvedTablesFilename: ~changefeed/resolved_tables.binpb
 	resolvedTablesFilename = jobInfoFilenamePrefix + "resolved_tables" + jobInfoFilenameExtension
+	// perTableProtectedTimestampsFilename: ~changefeed/per_table_protected_timestamps.binpb
```

Comment:
I realize that this isn't true for the resolved tables filename (maybe you can change that too?) but could you replace the underscores with hyphens? Seems like that's the convention in the codebase